### PR TITLE
Start cleaning up templates

### PR DESCRIPTION
**Pull request by imjuniper**
I'm starting to cleanup the markup for page templates, as well as convert their JavaScript over to Alpine. This is very much a work in progress, and with my 7 courses this semester, it might take a while to finish it. But at least now it will be a lot more readable, and will remove a lot of repetition and useless JS.

### CODE REVIEW

**Comment by jaapmarcus** [comment]
When ever possible I prefer smaller "merges" and more frequent then 1 or 2 very large merges :)

**Comment by imjuniper** [comment]
@jaapmarcus would you prefer that I do it one page at a time? In fact, it might make it easier for me too

**Comment by jaapmarcus** [comment]
> @jaapmarcus would you prefer that I do it one page at a time? In fact, it might make it easier for me too

That is fine for me.. (Or few at the time depending on size and what suits you)

We have done @AlecRust  massive css rewrite almost the same and it works fine.  (And still in progress) It also allows

**Comment by jaapmarcus** [comment]
@jakobbouchard  is this PR ready for review?

File: web/templates/pages/list_ssl.php [hunk 1]
```diff
@@ -2,48 +2,94 @@
...
-					<?= _("SSL Key") ?>
-					<a href="javascript:saveTextToBlob('<?php echo htmlentities($v_domain); ?>.key', 'v_key');"><i class="fas fa-download"></i></a>
+					<?=_("SSL Key");?>
+					<a
+						x-bind:href="blob()"
+						download="<?= htmlentities($v_domain) ?>.key"
+					><i class="fas fa-download"></i></a>
 				</label>
-				<textarea class="form-control u-min-height100" name="v_key" id="v_key"><?= $v_key ?></textarea>
+				<textarea
```
Comment (jaapmarcus) [inline web/templates/pages/list_ssl.php:57]: I am getting: 
Screenshot 2023-01-31 at 11 02 10
Screenshot 2023-01-31 at 11 00 55

**Review by jaapmarcus — CHANGES_REQUESTED** [review]
Also on Quick install clicking on generate will submit the form:

[Error] ReferenceError: Can't find variable: $
	Global Code (webapp:599)

**Comment by imjuniper** [comment]
@jaapmarcus I fixed both issues. The first one was due to certificates having newlines, so I base64 encode the value in the HTML, then decode it for the download blob and textarea value.

As for the second one, it was simply a mistake when changing empty (`href="#"`) links to buttons, some of them should have `type="button"` on them, since by default they submit the form at the same time as executing related code.

**Review by jaapmarcus — APPROVED** [review]

**Comment by jaapmarcus** [comment]
Works fine

**Pull request merged**